src/main/java/com/google/research/bleth/simulator/AbstractSimulation.java

```diff
@@ -94,10 +106,35 @@ void resolverEstimate() {
     void writeRoundState() { }
 
     /** Gather statistical data of the current round and update the aggregated simulation statistics based on all rounds. */
-    void updateSimulationStats() { }
+    void updateSimulationStats() {
```

Comment:
I would suggest calculating the empirical standard deviation of the distances as well.

Reply:
I don't want this PR to be too long, we can add it in another PR.